components/table/demo/table-test.js

```diff
@@ -99,7 +100,7 @@ class TestTable extends RtlMixin(DemoPassthroughMixin(TableWrapper, 'd2l-table-w
 					<tbody>
 						<tr class="d2l-table-header">
 							<th scope="col" sticky></th>
-							${this._renderDoubleSortButton('Avocado', 'Prune')}
+							${this._renderDoubleSortButton('Avocado')}
```

Comment:
We would also still want to keep this scenario. It's an important one to be able to test easily.